### DISCUSSION TITLE
rule: disallow `aws_launch_configuration` resource, prefer `aws_launch_template`

### DISCUSSION THREAD

**Discussion by drewmullen** [terraform-linters/tflint-ruleset-aws#420] [Ideas]
`aws_launch_template` is versioned and should be preferred over `aws_launch_configuration`

**Comment by drewmullen** [comment]
Is anyone willing to coach me through this rule creation or #304 ? I have a few other PRs id like to provide as well. I could also help with documenting from a new users perspective

**Comment by wata727** [comment]
You can create a rule from a template with `go run ./rules/generator`:
https://github.com/terraform-linters/tflint-ruleset-aws#add-a-new-rule

Please refer to the following documents for the SDK's API:
https://pkg.go.dev/github.com/terraform-linters/tflint-plugin-sdk/tflint#Runner
